docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -0,0 +1,266 @@
+# Migration from v5 to v6
+
+<p class="description">This guide describes the changes needed to migrate the Date & Time Pickers from v5 to v6.</p>
+
+## Start using the alpha release
+
+In `package.json`, change the version of the data grid package to `next`.
+
+```diff
+-"@mui/x-date-pickers": "latest",
++"@mui/x-date-pickers": "next",
+```
+
+Using `next` ensures that it will always use the latest v6 alpha release, but you can also use a fixed version, like `6.0.0-alpha.0`.
+
+## Breaking changes
+
+Since v6 is a major release, it contains some changes that affect the public API.
+These changes were done for consistency, improve stability and make room for new features.
+Below are described the steps you need to make to migrate from v5 to v6.
+
+### Update the format of the `value` prop
+
+The `value` prop of the pickers now expects a parsed value.
+Until now, it was possible to provide any format that your date management library was able to parse.
+For instance, you could pass `value={new Date()}` when using `AdapterDayjs`.
+
+This behavior brought a lot of confusion and was removed.
+The format expected by the `value` prop is now the same as for any other prop holding a date.
+Here is the syntax to initialize a date picker at the current date for each adapter:
+
+```tsx
+// Date-fns
+<DatePicker value={new Date()} />;
+
+// Dayjs
+import dayjs from 'dayjs';
+<DatePicker value={dayjs()} />;
+
+// Moment
+import moment from 'moment';
+<DatePicker value={moment()} />;
+
+// Luxon
+import { DateTime } from 'luxon';
+<DatePicker value={DateTime.now()} />;
+```
+
+### Rename the `LeftArrowButton` slot
+
+The component slot `LeftArrowButton` has been renamed `PreviousIconButton` on all pickers:
+
+```diff
+<DatePicker
+  components={{
+-   LeftArrowButton: CustomButton,
++   PreviousIconButton: CustomButton,
+  }}
+
+  componentsProps={{
+-   leftArrowButton: {},
++   previousIconButton: {},
+  }}
+/>
+```
+
+### Rename the `RightArrowButton` slot
+
+The component slot `RightArrowButton` has been renamed `NextIconButton` on all pickers:
+
+```diff
+<DatePicker
+  components={{
+-   RightArrowButton: CustomButton,
++   NextIconButton: CustomButton,
+  }}
+
+  componentsProps={{
+-   rightArrowButton: {},
++   nextIconButton: {},
+  }}
+/>
+```
+
+### Replace the `DialogProps` prop
+
+The `DialogProps` prop has been replaced by a `dialog` component props slot on all responsive and mobile pickers:
```

Comment:
What do you think about removing `slot` word for these cases to avoid confusion between `component slot`? 🤔 
```suggestion
The `DialogProps` prop has been replaced by a `dialog` component props on all responsive and mobile pickers:
```